src/backend/distributed/sql/citus--10.2-4--11.0-1.sql

```diff
@@ -80,3 +80,13 @@ BEGIN
       UPDATE pg_dist_node_metadata SET metadata=jsonb_set(metadata, '{partitioned_citus_table_exists_pre_11}', to_jsonb(partitioned_table_exists), true);
 END;
 $$;
+
+#include "udfs/run_command_on_all_nodes/11.0-1.sql"
+
+DROP VIEW IF EXISTS pg_catalog.citus_stat_activity CASCADE;
```

Comment:
These two seems unnecessary. How can we have these pre-11? If we are to follow this practice, we should do it for all types/views we create in upgrade scripts?
```SQL
DROP VIEW IF EXISTS pg_catalog.citus_stat_activity CASCADE;
DROP TYPE IF EXISTS pg_catalog.citus_stat_activity_one_node CASCADE;
```